lib/compile_time.flow

```diff
@@ -0,0 +1,34 @@
+export {
+	// Normally all these functions are eliminated from program before passing it to backends.
+
+	// Speculative function, used to ask to compute a constant value during compilation.
+	// In case a compile-time computation fails, the error will be reported
+	compileTime(v: ?) -> ? {
+		v
+	}
+
+	// Speculative function, used to ask to compute a constant value during compilation.
```

Comment:
When would we ever need this? A program is fixed. Either it works, or it does not.